src/detect.c

```diff
@@ -1150,7 +1149,8 @@ static bool DetectRunTxInspectRule(ThreadVars *tv,
                 TRACE_SID_TXS(s->id, tx, "engine %p match %d", engine, match);
                 if (engine->stream) {
                     can->stream_stored = true;
-                    can->stream_result = match;
+                    // TODO change Callback prototype ?
```

Comment:
yes I think that makes sense, also in a separate commit

Reply:
that makes a big commit...

Reply:
(done in next version)